### PR TITLE
Post a new comment to an existing article

### PR DESCRIPTION
**Pull request by yasarosmankhan**

### CODE REVIEW

File: src/components/Comments.jsx [hunk 1]
```diff
@@ -16,12 +21,35 @@ const Comments = () => {
...
 	}
 
 	return (
 		<>
+			<div>
+				<p></p>
+				{!username.length ? (
+					<div>
+						To comment on this post{' '}
+						<strong>
```
Comment (Augs0) [inline src/components/Comments.jsx:43]: What is this strong tag doing exactly?

File: src/components/LoginForm.jsx [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+
+	useEffect(() => {
+		getUsers().then((response) => {
+			setUsers(response.data.users);
+		});
+	}, []);
+
+	const handleSubmit = (e) => {
+		e.preventDefault();
+		const selectedValue = e.target.exampleFormControlSelect1.value;
```
Comment (Augs0) [inline src/components/LoginForm.jsx:22]: I think there must be a better name than exampleFormControlSelect1 😆

**Review by Augs0 — CHANGES_REQUESTED** [review]
Overall, I really like it Yasar, but there are some UX considerations you could make. When I post a comment, I expect to see something that tells me I was successful, otherwise I might try to post again. It might be nice to have this comment appear at the top too as it can be laborious scrolling down the whole list to find mine in a well-commented article. This is unrelated but I also realised I cause an error if I click like then dislike on an article 👀  so it's impossible to 'undo' my like, so to speak. Might be worth taking a look at that if you get time

**Comment by NRMSMITH** [comment]
I'd like to see some indication such as 'comment posted!' when it has been successful :) That way, if it's unsuccessful, I know to try again :)

**Comment by NRMSMITH** [comment]
Looks good to me! 👍

**Pull request merged**